tests/Go.Tests.ps1

```diff
@@ -57,22 +57,28 @@ Describe "Go" {
         Set-Location -Path $simpleLocation
         "go run simple.go" | Should -ReturnZeroExitCode
         "go build simple.go" | Should -ReturnZeroExitCode
-        "./simple" | Should -ReturnZeroExitCode
+        $compiledPackageName = "simple"
+        if ($IsWindows) { $compiledPackageName = "simple.exe" }
```

Comment:
`if ($IsWindows) { $compiledPackageName += ".exe" }`

Reply:
Thank you!